launch/camera_pipeline+machine.launch

```diff
@@ -1,12 +1,16 @@
 <launch>
 	<arg name="camera" default="camera"/>
 	<arg name="camera_id" default="0" />
+	<arg name="image_width" default="640" />
+	<arg name="image_height" default="480" />
 	<arg name="publish_rate" default="15"/>
 	<arg name="machine" default=""/>
 	<arg name="info_url" default=""/>
 
-	<node name="$(arg camera)" pkg="simple_webcam" type="camera.py" machine="$(arg machine)">
+	<node name="$(arg camera)" pkg="simple_webcam" type="camera.py" machine="$(arg machine)" output="screen" >
```

Comment:
The node prints an INFO statement that says what *actual* image resolution will be published. I forced printing because it might be a frustratingly transparent issue when you request a certain resolution via parameters but get a different one. You'd have to check the image width and height each time as you experiment with resolutions. That was my thought process; would you call that a non-debugging reason?